Content.Server/GameObjects/Components/Damage/DamageableComponent.cs

```diff
@@ -53,6 +67,8 @@ public override void ExposeData(ObjectSerializer serializer)
             {
                 Resistances = ResistanceSet.GetResistanceSet(name);
             });
+
+            serializer.DataField(ref _soundArray, "sounds", new Dictionary<DamageType, string> { });
```

Comment:
Can you use the SoundCollectionPrototype and pick a random sound?

Reply:
I'll add it to the TODO list and i'd like to support both a direct file and picking a random sound from a collection. I'll do my best, but i'm terrible at YAML serialisation so no promises.